### DISCUSSION TITLE
Need help understanding the output

### DISCUSSION THREAD

**Discussion by douginoz** [bemasher/rtlamr#282] [General]
I've bought a usb radio (NooElec NESDR Mini USB RTL-SDR & ADS-B Receiver Set, RTL2832U & R820T Tuner), and installed rtl_tcp and rtlamr in ubuntu 23.04.  
- The antenna is 10 feet from my meter (Itron C2SOD, listed as supported, and using 902.3 - 927.8 frequency).
-  I have 3 neighbours in about a 200' radius.  We're all on the same electricity provider (there's only one here, all houses built at the same time with the same meters).  BC Hydro, in BC Canada.

When I run rtlamr it displays some initial information and then nothing.  I didn't read anything anywhere that tells me if this is normal but I'm starting to get the idea that it can take hours to get a reading from some devices.  Several hours later I received a couple of lines, then again the next day.  But the results seem totally random and don't repeat.  So I can't deduce anything from them.  They don't correlate to any markings on my meter, and I find it hard to imagine it's picking up errant signals from 5 (not 3, but 5?) neighbours.  Or perhaps low-flying houses in random orbits circling the house?

 Here's the output.  I'd appreciate any feedback on what I'm seeing.

```
moa@sophie:~/docker_configs$ rtl_tcp -f 912000000
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001

Using device 0: Generic RTL2832U OEM
Detached kernel driver
Found Rafael Micro R820T tuner
[R82XX] PLL not locked!
Tuned to 912000000 Hz.
listening...
Use the device argument 'rtl_tcp=127.0.0.1:1234' in OsmoSDR (gr-osmosdr) source
to receive samples in GRC and control rtl_tcp parameters (frequency, gain, ...).
client accepted! localhost 37798
Allocating 15 zero-copy buffers
set freq 912600155
set sample rate 2359296
set gain mode 0


moa@sophie:~$ ./go/bin/rtlamr
20:31:00.939106 decode.go:45: CenterFreq: 912600155
20:31:00.939295 decode.go:46: SampleRate: 2359296
20:31:00.939308 decode.go:47: DataRate: 32768
20:31:00.939317 decode.go:48: ChipLength: 72
20:31:00.939326 decode.go:49: PreambleSymbols: 21
20:31:00.939336 decode.go:50: PreambleLength: 3024
20:31:00.939345 decode.go:51: PacketSymbols: 96
20:31:00.939355 decode.go:52: PacketLength: 13824
20:31:00.939366 decode.go:59: Protocols: scm
20:31:00.939376 decode.go:60: Preambles: 111110010101001100000
20:31:00.939386 main.go:124: GainCount: 29
{Time:2023-08-03T07:09:33.032 SCM:{ID:60657120 Type:14 Tamper:{Phy:02 Enc:02} Consumption: 6547332 CRC:0x6536}}
{Time:2023-08-03T12:31:21.932 SCM:{ID:13606219 Type: 0 Tamper:{Phy:02 Enc:01} Consumption: 2048697 CRC:0xEE2F}}
{Time:2023-08-04T02:10:15.393 SCM:{ID:42971139 Type:15 Tamper:{Phy:01 Enc:03} Consumption: 9959174 CRC:0x43E7}}
{Time:2023-08-04T15:19:00.530 SCM:{ID:58752433 Type: 7 Tamper:{Phy:03 Enc:01} Consumption:16607265 CRC:0x5148}}
{Time:2023-08-04T19:53:50.197 SCM:{ID:63235747 Type: 2 Tamper:{Phy:00 Enc:02} Consumption:  699189 CRC:0x9016}}
```

**Comment by bemasher** [comment]
It looks random because it probably is. If you listen to enough noise, eventually something will look enough like a packet to make it through.

The bad news is this means you likely don't have any supported meters in range.
